service/controller/v12/resource/instance/create.go

```diff
@@ -77,14 +74,48 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			}
 
 			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("set resource status to '%s/%s'", Stage, DeploymentInitialized))
-			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling reconciliation")
+
+			deploymentTemplateChk, err := getDeploymentTemplateChecksum(computedDeployment)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			if deploymentTemplateChk != "" {
+				err = r.setResourceStatus(customObject, DeploymentTemplateChecksum, deploymentTemplateChk)
+				if err != nil {
+					return microerror.Mask(err)
+				}
+
+				r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("set %s to '%s'", DeploymentTemplateChecksum, deploymentTemplateChk))
+			} else {
+				r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("Unable to get a valid Checksum for %s", DeploymentTemplateChecksum))
+				// todo remove any DeploymentTemplateChecksum leftovers from the CR
```

Comment:
What's this? Is this tracked somewhere?

FYI: https://github.com/giantswarm/fmt/blob/master/go/todo_comments.md

Reply:
this is harmless, I'll remove the todo